### DISCUSSION TITLE
Gross Volume units

### DISCUSSION THREAD

**Discussion by mgallardo15** [FIATA/eFBL#4] [Q&A] [answered]
Hi,

Is there any field where to indicate the Gross Volume for each Included Consignment Item?

Thank you
Manuel

**Comment by LuceliaTinembart** [comment, marked as answer]
Hi Manuel, 
thanks for your question. There is the **grossVolume** field which is required for each Consignment Item. 
https://fiata.github.io/eFBL/schema_doc.html#supply_chain_consignment_includedConsignmentItem_items_grossVolume

Hope this helps and answers your question?
Kind regards, 
Lucelia

**Comment by mgallardo15** [comment]
Hi Lucelia,

We already noticed there is a field for the grossVolume; the point is that we cannot specify which units we are using (m3, cm3, inches3, etc)

Regards
Manuel
________________________________
De: LuceliaT ***@***.***>
Enviado: lunes, 26 de abril de 2021 11:48
Para: FIATA/eFBL ***@***.***>
Cc: Manuel Gallardo ***@***.***>; Author ***@***.***>
Asunto: Re: [FIATA/eFBL] Gross Volume units (#4)


Hi Manuel,
thanks for your question. There is the grossVolume field which is required for each Consignment Item.
https://fiata.github.io/eFBL/schema_doc.html#supply_chain_consignment_includedConsignmentItem_items_grossVolume

Hope this helps and answers your question?
Kind regards,
Lucelia

**Reply by gdepourtales** [reply]
Hi Manuel

The `grossVolume` property is an object with two properties, namely `value` and `unit` See measure definition here: https://github.com/FIATA/eFBL/blob/main/json-schemas/definitions/classes/measure.schema.json. 

FIATA encourages the use of standard unit naming as defined in the CEFACT/ICG/2010/IC013 Recommendation N°. 20 - Revision 7. (Link here: https://unece.org/trade/uncefact/cl-recommendations) for the content of the `unit` property

**Comment by mgallardo15** [comment]
Hi Guy,

Thank you for your answer. It is indeed in the schema you send (we did not find it at the documentation, although we have just seen that the documentation states it is as in the grossWeight specification)
[cid:0f0aaf6e-cbfa-4f4a-ba55-9b592ed0c776]

Thank you again
Best regards
Manuel
________________________________
De: Guy de Pourtalès ***@***.***>
Enviado: lunes, 26 de abril de 2021 13:05
Para: FIATA/eFBL ***@***.***>
Cc: Manuel Gallardo ***@***.***>; Author ***@***.***>
Asunto: Re: [FIATA/eFBL] Gross Volume units (#4)


Hi Manuel

The grossVolume property is an object with two properties, namely value and unit See measure definition here: https://github.com/FIATA/eFBL/blob/main/json-schemas/definitions/classes/measure.schema.json.

FIATA encourages the use of standard unit naming as defined in the CEFACT/ICG/2010/IC013 Recommendation N°. 20 - Revision 7. (Link here: https://unece.org/trade/uncefact/cl-recommendations) for the cnotent of the unit property
